test/alphabet/aminoacid/aa27_test.cpp

```diff
@@ -0,0 +1,204 @@
+// ============================================================================
+//                 SeqAn - The Library for Sequence Analysis
+// ============================================================================
+//
+// Copyright (c) 2006-2017, Knut Reinert & Freie Universitaet Berlin
+// Copyright (c) 2016-2017, Knut Reinert & MPI Molekulare Genetik
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Knut Reinert or the FU Berlin nor the names of
+//       its contributors may be used to endorse or promote products derived
+//       from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+// LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+// OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+// DAMAGE.
+//
+// ============================================================================
+// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
+// ============================================================================
+
+#include <gtest/gtest.h>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/alphabet/aminoacid/aa27_sequence.hpp>
+
+using namespace seqan3;
+
+TEST(alphabet_aminoacid_aa27_test, default_ctr)
+{
+    constexpr aa27 amino{};
+
+    EXPECT_EQ(amino, aa27::A);
+    EXPECT_EQ(amino.value, aa27::c_type::A);
+}
+
+TEST(alphabet_aminoacid_aa27_test, copy_ctr)
+{
+    constexpr aa27 amino1{aa27::B};
+    constexpr aa27 amino2{amino1};
+
+    EXPECT_EQ(amino2, aa27::B);
+    EXPECT_EQ(amino2.value, aa27::c_type::B);
+}
+
+TEST(alphabet_aminoacid_aa27_test, move_ctr)
+{
+    constexpr aa27 amino1{aa27::C};
+    constexpr aa27 amino2 = std::move(amino1);
```

Comment:
this is not move construction, but move assignment! wrap in `{}` instead of using `=`